### ISSUE TITLE
Lifetime of "outbound-rtp"

### ISSUE THREAD

**Issue by henbos** [w3c/webrtc-stats#558]
The type definition for ["outbound-rtp"](https://w3c.github.io/webrtc-stats/#dom-rtcstatstype-outbound-rtp) says:
```
Statistics for an outbound <a>RTP stream</a> that is currently sent with this
{{RTCPeerConnection}} object.
```

But [The RTP statistics hierarchy](https://w3c.github.io/webrtc-stats/#the-rtp-statistics-hierarchy) says:
```
RTP monitored objects are not deleted.
```

The intent being not to delete "outbound-rtp" stats objects, the first paragraph should say "that is currently sent or has ever been sent by this RTCPeerConnection". This is a simple PR.

But note that the modern getStats() implementation in chromium does not implement this. And if it did...
- How can you tell which RTP streams are active and which are old? Do we need a new stat?
- What happens to trackId, senderId, mediaSourceId and transportId of an inactive rtp stream?

Alternatively, we could make the spec align with current implementations and say that they are deleted when the sender is reconfiured to send a different ssrc or if the sender stops sending.

**Comment by henbos** [comment]
Have we discussed this before @alvestrand ?

**Comment by henbos** [comment]
Add a clarification that they are not deleted but if not sending the timestamp is stale

**Comment by alvestrand** [comment]
We should:
1) Stop deleting stats
2) lastPacketSentTimestamp tells you if the track is on or off
3) we ought to have a boolean saying "BYE sent" (in which case sending more packets is illegal)

**Comment by henbos** [comment]
Suggestion: boolean for RTCP_BYE?

**Comment by alvestrand** [comment]
need a "received BYE" on inbound-rtp too.

**Comment by vr000m** [comment]
both at the sender and receiver. Sent bye vs received bye

**Comment by henbos** [comment]
Resolved in #667 and #668
